UndertaleModTool/Editors/UndertaleSpriteEditor.xaml.cs

```diff
@@ -193,5 +193,27 @@ private void MaskExport_Click(object sender, RoutedEventArgs e)
                 }
             }
         }
+
+        private void UndertaleObjectReference_Loaded(object sender, RoutedEventArgs e)
+        {
+            var objRef = sender as UndertaleObjectReference;
+
+            objRef.ClearRemoveClickHandler();
+            objRef.RemoveButton.Click += Remove_Click_Override;
+            objRef.RemoveButton.ToolTip = "Remove texture entry";
```

Comment:
```suggestion
            objRef.RemoveButton.ToolTip = "Remove texture entry.";
```

Reply:
All `UndertaleObjectReference` control tool tips don't have a dot, so I've written it also like that.

Reply:
Huh, could've sworn the tooltips have punctuation.